### DISCUSSION TITLE
Conversion Slider Mods

### DISCUSSION THREAD

**Discussion by Andreas010** [ppy/osu#21437] [Ideas] [answered]
I have an idea for two mods in the `Conversion` category for the default osu mode.
Beware, I am bad at naming things...

The first mod could be called "Sliderless". It takes every slider and creates two circles at the start and end, aswell as every tick. This would work simmilar to Mania's `Hold Off` mod.

The second mod could be called "Slide-here". It tries to make as much sliders as possible using combos. A mod config for it could be the spline type of the sliders. The mod would take a combo and connect all the circles using a slider and append already present ones. Behaviour for stacked circles could be creating a loop at that place, since pausing a slider just somewhere will throw people off.

**Comment by ghost** [comment, marked as answer]
> The first mod could be called "Sliderless". It takes every slider and creates two circles at the start and end, aswell as every tick. This would work simmilar to Mania's Hold Off mod.

Sounds like #20044 

> The second mod could be called "Slide-here". It tries to make as much sliders as possible using combos. 

I don't understand why you wouldn't just play relax. Either way, there's no tapping skill required and its all aim.
In addition, accelerating/decelerating sliders would be a pain to implement and play
